index.bs

```diff
@@ -11949,37 +11957,32 @@ the interfaces that |O| implements.
             and |prototype| has an own property named |P|, then return false.
         1.  Set |prototype| to be the value of the internal \[[Prototype]] property of |prototype|.
     1.  Return true.
-</div>
-
-<div class="note">
 
-    This should ensure that for objects with named properties, property resolution is done in the following order:
+    <div class="note">
```

Comment:
IMO this note should not be part of the algorithm, i.e., as it was before is better.